scrapers_next/va/committees.py

```diff
@@ -0,0 +1,149 @@
+from spatula import URL, CSS, HtmlListPage, HtmlPage, SelectorError, SkipItem
+from openstates.models import ScrapeCommittee
+import time
+
+# VA lists the full names of committee members on individual separate pages
+# MemberDetail grabs a member's full name from their specific page
+
+
+class MemberDetail(HtmlPage):
+    def process_page(self):
+        com = self.input[0]
+        role = self.input[1]
+        try:
+            mem_name = CSS("#mainC > h3").match(self.root)
+        except SelectorError:
+            return com
+
+        if "Delegate" in mem_name[0].text:
+            cleaned_name = mem_name[0].text.split("Delegate")[1].strip()
+        else:
```

Comment:
There's always a possibility that `Delegate` or `Senator` _won't_ be in the string. Perhaps `elif "Senator" in mem_name[0].text:`?

Reply:
good point! will change